src/meetingreadinesschecker/DefaultMeetingReadinessChecker.ts

```diff
@@ -46,8 +46,8 @@ export default class DefaultMeetingReadinessChecker implements MeetingReadinessC
 
   async checkAudioInput(audioInputDevice: Device): Promise<CheckAudioInputFeedback> {
     try {
-      await this.meetingSession.audioVideo.chooseAudioInputDevice(audioInputDevice);
-      await this.meetingSession.audioVideo.chooseAudioInputDevice(null);
+      await this.meetingSession.audioVideo.startAudioInput(audioInputDevice);
+      await this.meetingSession.audioVideo.startAudioInput(null);
```

Comment:
Suggestion: We've been using `undefined` as a default initial value. These can also accept `undefined` to choose none.

Reply:
Well, null for audio actually mean the empty device so I prefer to keep it that way :) we throw error for undefined value.